src/components/shared/Input/TokenInput.vue

```diff
@@ -143,6 +144,7 @@ export default class TokenInput extends Mixins(
   @Prop({ default: false, type: Boolean }) readonly external!: boolean;
   @Prop({ default: false, type: Boolean }) readonly loading!: boolean;
   @Prop({ default: false, type: Boolean }) readonly disabled!: boolean;
+  @Prop({ default: false, type: Boolean }) readonly sliderDisabled!: boolean;
```

Comment:
We don't need any additional prop, `isMaxAvailable` might be re-used

Reply:
The component is general-purpose, i wouldn't mess logic up
there are cases when max is not available but slider should function - when entered amount is equal to max amount (the max should be hidden, but slider should be access-able)